pubspec.lock

```diff
@@ -729,10 +729,10 @@ packages:
     dependency: transitive
     description:
       name: google_maps_flutter_web
-      sha256: f893d1542c6562bc8299ef768fbbe92ade83c220ab3209b9477ec9f81ad585e4
+      sha256: "05067c5aa762ebee44b7ef4902a311ed8cf891ef655e2798bae063aa3050c8d9"
       url: "https://pub.dev"
     source: hosted
-    version: "0.5.4+2"
+    version: "0.5.4+1"
```

Comment:
Why did you downgrade this?

Reply:
I did not intentionally downgrade any packages. The downgrade occurred automatically right after I updated the `reaction_button` package in the `pubspec.yaml` file.

Reply:
fixed the package downgrading issue.